test/features/navigation.feature

```diff
@@ -0,0 +1,17 @@
+@smoke
+@navigation
+Feature: Navigation
+
+
+Scenario Outline: Should scroll to "<navigationLink>"
+    Given I open pzz.by
+    When I navigate to "<navigationLink>" using navigation menu
+    And I wait 1 seconds
```

Comment:
Please remove this line from the scenario.

Reply:
removed